### ISSUE TITLE
Polyon rendering glitch

### ISSUE THREAD

**Issue by springmeyer** [maplibre/maplibre-tile-spec#221]
I noticed in the demo instance that @ebrelsford put up at https://mltdemo.stamen.com that there is some glitch art in Antarctic:

<img width="437" alt="Screenshot 2024-06-28 at 11 42 48 AM" src="https://github.com/maplibre/maplibre-tile-spec/assets/20300/4f69c8f7-71e9-44ed-8e3e-b1d9077e1893">

<img width="329" alt="Screenshot 2024-06-28 at 11 43 04 AM" src="https://github.com/maplibre/maplibre-tile-spec/assets/20300/54374df1-2d4e-488e-b31e-6cf92eac4750">

@ebrelsford how recently where the MLTs used in the demo encoded? Can you make sure to re-encode them with a java encoder version that includes https://github.com/maplibre/maplibre-tile-spec/pull/193? Is the glitch still present?

**Comment by springmeyer** [comment]
I also noticed there are some subtle tile boundary artifacts in the center of Africa:

<img width="487" alt="Screenshot 2024-06-28 at 11 45 13 AM" src="https://github.com/maplibre/maplibre-tile-spec/assets/20300/174679ea-7321-4f7c-bc51-f6281f5f01e1">

I'm almost positive these should be fixed by re-encoding after #193.

**Comment by ebrelsford** [comment]
I encoded these this morning so they should include the changes from #193 but I can try again, there's a chance I was on a wrong branch.

**Comment by ebrelsford** [comment]
On `main` I rebuilt the cli (`./gradlew cli`) and am running `encode.jar` on the input MVTs, seeing the same glitches.

**Comment by springmeyer** [comment]
Okay, thanks for checking.

**Comment by louwers** [comment]
Closing, since a lot of changes where made since this issue was created.

Please create a new issue with a reproducer in case similar issues are still present.
